pkg/cri/server/container_start.go

```diff
@@ -178,6 +183,8 @@ func (c *criService) StartContainer(ctx context.Context, r *runtime.StartContain
 	c.eventMonitor.startContainerExitMonitor(context.Background(), id, task.Pid(), exitCh)
 
 	containerStartTimer.WithValues(info.Runtime.Name).UpdateSince(start)
+	span.AddEvent("container started",
+		tracing.SpanEventAttribute(tracing.Attribute("container.start.duration", time.Since(start).String())))
```

Comment:
Isn't it the same as the duration of the span?

Reply:
Since, I am using `tracing.SpanFromContext(ctx)`, I cannot be sure what the original span is coming from and hence the duration here might be useful.